recipes/sentry-crashpad/all/conanfile.py

```diff
@@ -68,6 +68,8 @@ def validate(self):
         elif tools.Version(self.settings.compiler.version) < minimum_version:
             raise ConanInvalidConfiguration("Build requires support for C++14. Minimum version for {} is {}"
                 .format(str(self.settings.compiler), minimum_version))
+        if tools.Version(self.version) < "0.4" and self.settings.os == "Macos" and self.settings.arch == "armv8":
+            raise ConanInvalidConfiguration("This version doesn't support ARM compilation".format(self.version))
```

Comment:
Only apply this if c3i fails again.
```suggestion
            raise ConanInvalidConfiguration("This version doesn't support ARM compilation")
```